<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SecureDlgBase</class>
  <widget class="QDialog" name="SecureDlg" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>415</width>
        <height>151</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Secure chanel request</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item>
        <widget class="QLabel" name="lblText" >
          <property name="text" stdset="0" >
            <string>Secure channel is established using SSL with Diffie-Hellman key exchange and  the TLS version 1 protocol.&#xd;
&#xd;
This only works with other SIM and Licq clients</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lblError" >
          <property name="font" stdset="0" >
            <font>
              <bold>true</bold>
            </font>
          </property>
          <property name="text" stdset="0" >
            <string/>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="btnCancel" >
              <property name="text" stdset="0" >
                <string>&amp;Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer3" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
